deps.bzl

```diff
@@ -30,9 +30,10 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive", "http_file"
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
 rules_scala_version = "0f89c210ade8f4320017daf718a61de3c1ac4773"
-rules_haskell_version = "f1323829e7e7cc1ea71013efe22466a6298a59b2"
-rules_haskell_sha256 = "addd6b4d8d63fd5c3ed1a4a2a72f071397203474dbb9a9fe0487c917ac219d55"
-rules_nixpkgs_version = "a169f54bfc48ad3ade9f46acac9fae7d493ad94b"
+rules_haskell_version = "b8f4b6d8f3847edbcfd122f633e3a90e7ec9349d"
```

Comment:
Can be pointed to master once https://github.com/tweag/rules_haskell/pull/1143 is merged.